src/Frozennode/Administrator/AdministratorServiceProvider.php

```diff
@@ -98,7 +103,7 @@ public function register()
 		$this->app['admin_datatable'] = $this->app->share(function($app)
 		{
 			$dataTable = new DataTable($app->make('itemconfig'), $app->make('admin_column_factory'), $app->make('admin_field_factory'));
-			$dataTable->setRowsPerPage($app->make('session.store'), Config::get('administrator::administrator.global_rows_per_page'));
+			$dataTable->setRowsPerPage($app->make('session.store'), Config::get('administrator.global_rows_per_page'));
 
 			return $dataTable;
 		});
```

Comment:
it might be better for users to register it in their app config, some people might already have it registered their, wouldn't want any conflict. I updated the readme to include this.